pkgs/development/libraries/fontconfig/default.nix

```
@@ -86,15 +89,23 @@ stdenv.mkDerivation (finalAttrs: {
       ${./make-fonts-conf.xsl} $out/etc/fonts/fonts.conf \
       > fonts.conf.tmp
     mv fonts.conf.tmp $out/etc/fonts/fonts.conf
-    # We don't keep section 3 of the manpages, as they are quite large and
-    # probably not so useful.
-    rm -r $bin/share/man/man3
   '';
 
-  passthru.tests = {
-    pkg-config = testers.hasPkgConfigModules {
-      package = finalAttrs.finalPackage;
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  doInstallCheck = true;
+  versionCheckProgram = "${placeholder "bin"}/bin/fc-list";
+  versionCheckProgramArg = "--version";
+
+  passthru = {
+    tests = {
+      pkg-config = testers.hasPkgConfigModules {
+        package = finalAttrs.finalPackage;
+      };
     };
+
+    updateScript = nix-update-script { };
```

`nix-update` heuristics might not handle the dist tarball but the following should work:

```suggestion
    updateScript = gitUpdater {
      url = "https://gitlab.freedesktop.org/fontconfig/fontconfig.git";
    };
```